### PR TITLE
Commit Search skeleton

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) commits the search project skeleton to the integration branch. Topics are empty, but files created per the current draft. TOC updated along with menu scopes too.

## Magento release version

- [ ] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [x] Other feature set, please specify: Search integration

### CODE REVIEW

**Review by leslietilling — CHANGES_REQUESTED** [review]
I'd like to omit the `ls` from the topic file names and not repeat the keywords in the URL structure.

Let's make the intro topic: live-search/overview.md

And other topics can simply remove the `ls`, such as live-search/data-types.md

File: src/live-search/live-search.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
```
Comment (leslietilling) [inline src/live-search/live-search.md:1]: This one also repeats in the URL. It's basically a mini landing page. 

maybe `live-search/sections` ?  We should come up with a standard for this kins of thing. Some of our existing URLs need work.

Comment (hickskenh) [reply src/live-search/live-search.md:1]: i used the same format as SEO and Search
https://docs.magento.com/user-guide/marketing/seo-search.html

Comment (leslietilling) [reply src/live-search/live-search.md:1]: Leave for now, but will have to figure out before we publish.

**Review by leslietilling — APPROVED** [review]

**Pull request merged**